Honor LDFLAGS, and use CXXFLAGS when linking so that the right C++
standard library is used.
https://sourceforge.net/p/xdiskusage/bugs/25/
--- Makefile.orig	2015-02-09 17:12:19.000000000 -0600
+++ Makefile	2018-12-07 10:15:52.000000000 -0600
@@ -17,7 +17,7 @@
 all:	$(PROGRAM)
 
 $(PROGRAM) : $(OBJECTS)
-	$(CXX) -o $(PROGRAM) $(OBJECTS) `fltk-config --ldflags`
+	$(CXX) -o $(PROGRAM) $(OBJECTS) `fltk-config --ldflags` $(CXXFLAGS) $(LDFLAGS)
 
 configure: configure.in
 	autoconf
--- makeinclude.in.orig	2014-11-20 15:20:11.000000000 -0600
+++ makeinclude.in	2018-12-07 10:06:58.000000000 -0600
@@ -22,4 +22,6 @@
 CXXFLAGS_D	=@CXXFLAGS_D@
 
 # libraries to link with:
+LDFLAGS	=@LDFLAGS@
+
 INSTALL	=@INSTALL@
